src/runtime/types/index.d.ts

```diff
@@ -1,13 +1,79 @@
-export type DirectusUser = object | null;
+type ArrayOfOrSingle<T> = T | T[];
+
+export type DirectusUser = {
```

Comment:
Having interfaces instead of types allow augmenting them while consuming the library: https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces

I think it's better to keep interfaces everywhere we can

Reply:
Can you give an example situation where `interface` would be benificial over `type`?